packages/lit-dev-content/site/docs/v3/releases/upgrade.md

```diff
@@ -1,40 +1,64 @@
 ---
-title: Upgrade guide
+title: Upgrade to Lit 3.x guide
 eleventyNavigation:
-  key: Upgrade guide
+  key: Upgrade to Lit 3.x guide
   parent: Releases
   order: 3
 versionLinks:
   v2: releases/upgrade/
 ---
 
-## Overview
+<div class="alert alert-info">
+
+If you are looking to migrate from Lit 1.x to Lit 2.x, see the [Lit 2.0 upgrade guide](/docs/v2/releases/upgrade/).
 
-Lit 3.0 has very few breaking changes from Lit 2.0. Any projects using Lit 2
-without deprecation warnings should not require any changes to upgrade to Lit 3.
+</div>
 
-Lit 3.0:
+## Overview
 
-- Drops support for IE11
-- Is published as ES2021
-- Removes a few deprecated APIs
+Lit 3.0 has very few breaking changes from Lit 2.0:
+
+- IE11 is no longer supported.
+- Lit's npm modules are now published as ES2021.
+- APIs deprecated with the Lit 2.0 release have been removed.
+- SSR hydration support modules have moved to the `@lit-labs/ssr-client` package.
+- Decorator behavior has been unified between TypeScript experimental decorators and standard decorators.
+- Support was removed for Babel decorators version "2018-09"
 
 Lit 3.0 should require no changes to upgrade from Lit 2.0 for the vast majority
 of users. Most apps and libraries should be able to extend their npm version
 ranges to include both 2.x and 3.x, like `"^2.7.0 || ^3.0.0"`.
 
-Lit 2.x and 3.0 are _interoperable_: templates, base classes, directives,
-decorators, etc., from one version of Lit will work with those from another.
+Lit 2.x and 3.0 are _interoperable_: templates, base classes, and directives from one version of Lit will work with those from another.
+
+## Decorator behavior has been unified
+
+To make the upgrade path from experimental decorators to standard decorators as
+smooth as possible, existing experimental decorators now support the standard
```

Comment:
WDYT about these words:

> In Lit 3.0 our decorators support running in three configurations: 
>
> 1. With the TypeScript `experimentalDecorators` flag and properties without the `accessor` keyword. This is backwards compatible with Lit 2.0 users. As standard decorators are still immature, this the most performant option that we recommend for for production for the time being.
> 2. With the TypeScript `experimentalDecorators` flag and with the `accessor` keyword. This aids migration to standard decorators. Code can add the `accessor` keyword one property at a time to migrate gradually.
> 3. With TypeScript 5.2 and without the `experimentalDecorators` flag, or with Babel 7.23.0, and with the `accessor` keyword.
>
> Each of these configurations is self contained. An application can mix and match libraries written in each of these configurations, and it is not generally a breaking change for a library to switch from one decorator configuration to another.

Reply:
I like it! Added it. I think this also includes the information that was proposed in the other comment. Namely that TS 5.2 and Babel 7.23.0 are required.